scripts/Testing/Testcases/config_tests.xml

```diff
@@ -18,18 +18,38 @@
     <REST_OPTION>none</REST_OPTION>
     <INFO_DBUG>1</INFO_DBUG>
     <CCSM_TCOST>4</CCSM_TCOST>
-    <DOUT_S>FALSE</DOUT_S> 
+    <DOUT_S>FALSE</DOUT_S>
   </test>
 
   <test NAME="ERI">
```

Comment:
Fairly.   We may need to have some code for individual tests in the system_test.py module but I haven't found the case that requires it yet.